gadget/layout.go

```diff
@@ -73,13 +73,12 @@ type PartiallyLaidOutVolume struct {
 	LaidOutStructure []LaidOutStructure
 }
 
-// LaidOutStructure describes a VolumeStructure that has been placed within the
-// volume
+// LaidOutStructure describes an OnDiskStructure that has been placed within the
+// volume.
```

Comment:
I think the old doc comments conceptually is still correct, it could be expanded though